src/main/rx/rx.c

```diff
@@ -75,7 +75,6 @@ static timeUs_t lastMspRssiUpdateUs = 0;
 
 rssiSource_t rssiSource;
 
-static bool rxDataReceived = false;
```

Comment:
With suggested changes this does not need to be a member of `rxRuntimeConfig_t`.